src/_lib/tzTokenizeDate/index.js

```diff
@@ -43,16 +43,43 @@ function hackyOffset(dtf, date) {
 var dtfCache = {}
 function getDateTimeFormat(timeZone) {
   if (!dtfCache[timeZone]) {
-    dtfCache[timeZone] = new Intl.DateTimeFormat('en-US', {
+    // @see https://github.com/marnusw/date-fns-tz/issues/38
+    var testDateTime = new Intl.DateTimeFormat('en-US', {
       hour12: false,
-      timeZone: timeZone,
+      timeZone: 'America/New_York',
       year: 'numeric',
       month: '2-digit',
       day: '2-digit',
       hour: '2-digit',
       minute: '2-digit',
-      second: '2-digit'
-    })
+      second: '2-digit',
+    });
+
+    var testDateTimeFormatted = testDateTime.format(new Date("2014-06-25T04:00:00.123Z"))
+    if(testDateTimeFormatted === '06/25/2014, 00:00:00' || testDateTimeFormatted === '‎06‎/‎25‎/‎2014‎ ‎00‎:‎00‎:‎00') {
```

Comment:
@hyyan it doesn't show here, but in my IDE I see strange characters before and after each part of the second date string you are comparing with. What character is this, and is it there intentionally?

I'm also curious why it is possible to have formatted output either with or without the comma? I would have expected only one possibility for a given locale.

Reply:
I noticed when using the timezone polyfill for IE11, the date gets formatted without a comma, that's why I added the check, the string which contains hidden characters is copied directly from IE11 console